addons/event/static/src/client_action/event_registration_summary_dialog.js

```diff
@@ -46,15 +47,15 @@ export class EventRegistrationSummaryDialog extends Component {
         }
 
         onMounted(() => {
-            if (this.props.registration.status === 'already_registered' || this.props.registration.status === 'need_manual_confirmation') {
+            if ((this.props.registration.status === 'already_registered' || this.props.registration.status === 'need_manual_confirmation') && this.props.playSound) {
                 this.props.playSound("notify");
-            } else if (this.props.registration.status === 'not_ongoing_event' || this.props.registration.status === 'canceled_registration') {
+            } else if ((this.props.registration.status === 'not_ongoing_event' || this.props.registration.status === 'canceled_registration') && this.props.playSound) {
```

Comment:
['already_registered', 'need_manual_confirmation'].includes(this.props.registration.status) to gain a bit of space? :)

Reply:
same for the other of course